yogstation/code/game/objects/items/bandage.dm

```diff
@@ -0,0 +1,147 @@
+	//handles bandage healing per tick, called in life
+	if (!used)
+		if (healing_limb && healing_limb.status == BODYPART_ORGANIC)
+			var/success = FALSE
+			switch (healtype)
+				if ("brute")
+					if (healing_limb.brute_dam)
+						success = healing_limb.heal_damage(healamount/src.duration, 0, 0)
+					else
+						to_chat(H, "<span class='notice'>The wounds on your [src.healing_limb.name] have stopped bleeding and appear to be healed.</span>")
+						used = TRUE
+				if ("burn")
+					if (healing_limb.burn_dam)
+						success = healing_limb.heal_damage(0, healamount/src.duration, 0)
+					else
+						used = TRUE
+						to_chat(H, "<span class='notice'>The burns on your [src.healing_limb.name] feel much better, and seem to be completely healed.</span>")
+			if (success)
+				H.update_damage_overlays(0)
+			if (staunch_bleeding && !H.bleedsuppress)
+				H.suppress_bloodloss(staunch_bleeding)
+			if (activefor <= src.duration)
+				activefor += 1
+			else
+				used = TRUE
+	else
+		//eject the bandage onto the floor with
+		fall_off(H, healing_limb)
+
+/obj/item/medical/bandage/proc/unwrap(mob/living/M, mob/living/carbon/human/T)
+	//DUPLICATE CODE BUT I'M FUCKING LAZY <- this was not Morrow
+	if (healing_limb.bandaged)
+		M.visible_message("<span class='warning'>[M] grabs and pulls at the [src] on [T]'s [src.healing_limb.name], unwrapping it instantly!</span>", "<span class='notice'>You deftly yank [src] off [T]'s [src.healing_limb.name].</span>")
+		name = "used [src.name]"
+		desc = "Piled into a tangled, crusty mess, these bandages have obviously been used and then disposed of in great haste."
+		color = "red"
+		loc = T.loc
+		healing_limb.bandaged = FALSE
+		used = TRUE
+
+
+/obj/item/medical/bandage/proc/fall_off(mob/living/carbon/human/H, obj/item/bodypart/L)
+	if (L.bandaged)
+		to_chat(H, "You loosen the bandage around [L.name] and let it fall to the floor.")
+		name = "used [src.name]"
+		desc = "Bloodied and crusted, these bandages have clearly been used and aren't fit for much anymore. Seems as if they were wrapped around someone's [L.name] last."
+		color = "red"
+		loc = H.loc
+		L.bandaged = FALSE
+		used = TRUE
+
+/obj/item/medical/bandage/proc/apply(mob/living/user, mob/tar, obj/item/bodypart/lt)
+	if (!ishuman(user))
+		to_chat(user, "<span class='warning'>You don't have the dexterity to use this!</span>")
+		return 0
+
+	if (ishuman(tar))
+		if (!lt.bandaged)
+			if (user == tar)
+				user.visible_message("<span class='notice'>[user] begins winding [src] about their [lt.name]..</span>", "<span class='notice'>You begin winding [src] around your [lt.name]..</span>")
+			else
+				user.visible_message("<span class='notice'>[user] begins winding [src] about [tar]'s [lt.name]..</span>", "<span class='notice'>You begin winding [src] around [tar]'s [lt.name]..</span>")
+
+			if (do_after(user, 50, target = tar))
+				if(!user.canUnEquip(src))
+					return 0
+				healing_limb = lt
+				lt.bandaged = src
+				moveToNullspace(src)
```

Comment:
You should just qdel it

Reply:
Qdel would kill the variables if I understand it correctly so it would fuck it all up.  Obviously could be wrong.